docs/conf/portland/2024/cfp-email-templates.rst

```diff
@@ -228,23 +228,56 @@ Subject:
 
    Hi {name},
 
-   We're just over a month away from meeting up in Portland! I hope you're feeling relaxed and confident.
+   We're just over a month away from meeting up in {{city}}! I hope you're feeling relaxed and confident.
 
-   The conference schedule is [live](https://www.writethedocs.org/conf/portland/2024/schedule/#monday-may-8)! :-)
+   The conference schedule is [live](https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/schedule/)! :-)
 
    I've got a couple more small logistical checkboxes for you:
 
-   ○ please register your free [speaker ticket]({{cfp_variables['ticket']}}), it includes hoodies sizing for a conference hoodie.
+   ○ please register your free [speaker ticket]({{cfp_variables['ticket']}}).
 
-   ○ please make sure to log in to [Pretalx](https://pretalx.com/write-the-docs-portland-2024/login/) and fill out the questions on pronouns, name pronunciation and interests for our emcee intro.
+   ○ please make sure to log in to [Pretalx](https://pretalx.com/wtd-{{shortcode}}-{{year}}/me/) and fill out the questions on pronouns, name pronunciation and interests for our emcee intro. We also need your **hoodie size** for a free speaker hoodie!
```

Comment:
Should we remove hoodie size from the speaker ticket?

Reply:
Yes, probably, though we should revert next year. :-D Do you mind doing that?

Reply:
Done.